File: C++/1752. Check if Array Is Sorted and Rotated.cpp
class Solution {
public:
    bool check(vector<int>& n) {
        int c=0;
        for(int i=0;i<n.size()-1;i++){
            if(n[i]>n[i+1])
            c++;
        }
        if(n[0]<n.back())
        c++;
        return c<2;
    }
};

File: JAVA/1752. Check if Array Is Sorted and Rotated.java
class Solution {
    public boolean check(int[] n) {
        int c=0;
        for(int i=0;i<n.length-1;i++){
            if(n[i]>n[i+1])
            c++;
        }
        if(n[0]<n[n.length-1])
        c++;
        return c<=1;
    }
}